Constants/index.js

```diff
@@ -0,0 +1,15 @@
+const validate = {
+  MONTHLYINVESTMENTMIN: 500,
+  MONTHLYINVESTMENTMAX: 100000,
+  INVESTMENTPERIODMIN: 1,
+  INVESTMENTPERIODMAX: 30,
```

Comment:
why these 1,30 declaration is repetedly make only two and use for every case and naming should be like this INVESTMENT_PERIOD_MAX

Reply:
okay sir... but if we make rateofreturn and delay max control by single variable, suppose in future if  we want to change limit for only delay then we need to introduced a new variable.

Reply:
yes